app/src/main/java/http_server/Router.java

```diff
@@ -0,0 +1,14 @@
+package http_server;
+import http_server.routes.HelloWorld;
+
+public class Router {
+
+  public String routeRequest(Request request) {
+    if ("/simple_get_with_body".equals(request.getPath())) {
+      HelloWorld body = new HelloWorld();
+      return body.getHelloWorld();
+    }
+
```

Comment:
Just as a reminder, when you see three things that have the same pattern, then try refactoring to dry it up. This gives you enough examples to know what pattern to use. Ex: Routing